src/math/Stack.cc

```diff
@@ -260,6 +265,11 @@ void statisticsStack(image::MaskedImage<PixelT> &out,
                                                                 wvector);  // use wvector
         }
     } else {
+        if (!wvector.empty()) {
+            LOGLS_WARN(_log,
+                       "Weights passed on to statisticsStack are ignored as sctrl.getWeighted() is False."
+                       "Set sctrl.setWeighted(True) for them to be used.");
+        }
```

Comment:
You can just use `LOG_WARN` here.  The `LOGL_` and `LOGLS_` are just different ways of doing string formatting on the message, but you're not doing any here.

Reply:
The use of LOG_WARN appears pretty minimal, with no occurrence except in its own unit test. Furthermore, I'm not sure if we want this to go into a default logger if there is one since it appears multiple times. So, we use LOGL_WARN and log it into the appropriate log file instead.

Reply:
Sorry, I should have suggested `LOGL_WARN`, not `LOG_WARN`; I hadn't realized that the "L" suffix controlled whether a log object was accepted and only the "S" suffix controlled the string formatting.  But what you've merged is fine.